src/frontend/Komponenter/Behandling/VedtakOgBeregning/Barnetilsyn/VedtakOgBeregningBarnetilsyn.tsx

```diff
@@ -64,11 +70,20 @@ const VedtakOgBeregningBarnetilsyn: FC<Props> = ({ behandling, vilkår }) => {
                                     behandling={behandling}
                                     alleVilkårOppfylt={alleVilkårOppfylt}
                                     ikkeOppfyltVilkårEksisterer={true}
-                                    lagretVedtak={vedtak}
+                                    lagretVedtak={
+                                        vedtak?._type === IVedtakType.Avslag ? vedtak : undefined
+                                    }
```

Comment:
Denne sjekken er egentlig ikke nødvendig siden den gjøres i `AvslåVedtak.tsx` men man kunne endret i den filen så den tar inn `lagretVedtak: IAvslagVedtak` i stedet for `IVedtakForOvergangsstønad`

Reply:
Vi endret på det fordi det ga feilmelding, men nå blir det dobbeltsjekk. Så enten holde det her eller bytte i AvslagVedtak til å være `IVedtak`